Thrive.csproj

```diff
@@ -522,6 +523,7 @@
     <Compile Include="src\general\GameWorld.cs" />
     <Compile Include="src\general\WorldGenerationSettings.cs" />
     <Compile Include="src\general\DifficultyPreset.cs" />
+    <Compile Include="src\general\ScreenEffect.cs" />
```

Comment:
I think this should be in the "engine" folder as I think fullscreen colour effects should probably be considered an "engine feature".